scrapers/HobbyPorn.yml

```
@@ -0,0 +1,39 @@
+name: HobbyPorn
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - hobby.porn
+    scraper: sceneScraper
+sceneByQueryFragment:
+  action: scrapeXPath
+  queryURL: "{url}"
+  scraper: sceneScraper
+sceneByName:
+  action: scrapeXPath
+  queryURL: https://hobby.porn/search/{}/
+  scraper: sceneSearch
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title: //div[@class="title-block"]/h1/text()
+      Performers:
+        Name: //span[@itemprop="author"]/a/text()
+      Studio:
+        Name:
+          fixed: Pornhub
```

This doesn't seem quite right, but the since the site lists the studio (or "author") along with the performers we might need to hack around that

For this I used a [German Scout scene](https://hobby.porn/video/slim-dreadlocks-girl-nicky-pickup-and-public-fuck-at-street-casting/) (which has a studio and a performer) and a [Morgpie scene]() (where Morgpie is both the "studio" and the performer)

```suggestion
      Performers:
        Name: //span[@itemprop="author" or @itemprop="actor"]/a
      Studio:
        Name: //span[@itemprop="author"]/a
```

This will unfortunately lead to the studio being listed as a performer, but as far as I can tell there's no way to avoid that given the site layout